operators/bosh-operator/DirectorService.js

```diff
@@ -849,7 +848,7 @@ class DirectorService extends BaseDirectorService {
       })
       .then(resource => {
         let response = _.get(resource, 'status.response', undefined);
-        if (response) {
+        if (!_.isEmpty(response)) {
```

Comment:
Just curious, why did we add this `_.isEmpty` here. In what situations we will have empty `response` object?